src/main/resources/templates/application/proceedings-remove.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>LAA CCMS - Start new application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/application/application/proceedings-and-costs}" class="govuk-back-link">Cancel and return to proceeding and costs</a>
```

Comment:
Should be plural proceedings in the link text

Reply:
amended